### ISSUE TITLE
bug: Ionic V4 - Platform.is not detecting ipad

### ISSUE THREAD

**Issue by JerryBels** [ionic-team/ionic#18838]
# Bug Report

**Ionic version:**
[x] **4.1.1**

**Current behavior:**
On an Ipad pro 12.9'' or an Ipad air 2 emulators, `platform.platforms()` returns : 
`iphone,ios,cordova,mobile,hybrid`
Also, as expected then, `platform.is('ipad')` returns false.

**Expected behavior:**
Should return `ipad` and `tablet` and no `iphone` in the list, and the `is()` check should then return true.

**Steps to reproduce:**
From an ipad emulator:
`console.log(platform.platforms());`

**Other information:**

**Ionic info:**

```
$ ionic info

Ionic:

   ionic (Ionic CLI)             : 4.12.0 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.1.1
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.3.9
   @angular/cli                  : 7.3.9
   @ionic/angular-toolkit        : 1.5.1

Cordova:

   cordova (Cordova CLI) : 9.0.0
   Cordova Platforms     : android 7.1.4, ios 5.0.0
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.0.1, (and 15 other plugins)

System:

   ios-deploy : 1.9.4
   ios-sim    : 8.0.1
   NodeJS     : v10.15.3 (/usr/local/bin/node)
   npm        : 6.9.0
   OS         : macOS Mojave
   Xcode      : Xcode 10.2.1 Build version 10E1001
```

**Comment by liamdebeasi** [comment]
Thanks for the issue. This was fixed in Ionic 4.1.2: https://github.com/ionic-team/ionic/blob/master/CHANGELOG.md#412-2019-03-20

I am going to close this as the issue has already been resolved. If the issue show up with the latest version of Ionic, feel free to open a new issue. Thanks!

**Comment by JerryBels** [comment]
Okay, thanks for your help, will try updating and see what goes.
